### DISCUSSION TITLE
Update for the video to text model

### DISCUSSION THREAD

**Discussion by devkatalist** [SubtitleEdit/subtitleedit#5893] [Q&A] [answered]
Hello. After downloading the video to text model for french for example, is it automatically updated when the zamia-speech repo is, or do we need to manually check for changes?

**Comment by niksedk** [comment, marked as answer]
Hi, there's no automatic updating, sorry.

**Reply by devkatalist** [reply]
OK. Thanks. Also, I just thought I should point out that there is a more powerful engine in their website that more accurate but also heavier (1.4 G). I've downloaded it and it works perfectly on SE. So thanks for that as well.

**Reply by niksedk** [reply]
Yes, I've added the heavy models to the UI download too :)
